docs/how_to/error-mitigation.rst

```diff
@@ -144,7 +144,8 @@ The Estimator interface lets users seamlessly work with the variety of error mit
       session.close() 
 
 .. note::
-    As you increase the resilience level, you will be able to use additional methods to improve the accuracy of your result. However, because the methods become more advanced with each level, they require additional sampling overhead (time) to generate more accurate expectation values.     
+    As you increase the resilience level, you will be able to use additional methods to improve the accuracy of your result. However, because the methods become more advanced with each level, they require additional sampling overhead (time) to generate more accurate expectation values.
+    Note that higher resilience levels do not guarantee better quality. Higher levels only mean greater overhead. Each method has its strengths and weaknesses. For example, TREX (Twirled Readout Error eXtinction) is good for shallow circuits since it's readout error mitigation whereas ZNE (Zero Noise Extrapolation) for deeper circuits. PEC can mitigate arbitrary errors but may not work in practice because of its large overhead.
```

Comment:
```suggestion
    Note that higher resilience levels do not guarantee better quality. Higher levels only mean greater overhead. Each method has its strengths and weaknesses. For example, TREX (Twirled Readout Error eXtinction) is good for shallow circuits because of its readout error mitigation whereas ZNE (Zero Noise Extrapolation) is good for deeper circuits. PEC can mitigate arbitrary errors but may not work in practice because of its large overhead.
```

Reply:
Thank you very much!